src/main/scripts/src/app/shared/search-box/search-box.component.html

```diff
@@ -12,6 +12,6 @@
   </div>
 </div>
 
-<button (click)="onButtonClick()">
+<button class="fa fa-search fa-fw" (click)="onButtonClick()">
```

Comment:
Although it's basically the same, isn't it better the way it is? The way I see the difference between the two options is: 
- The button is the magnifying glass.
- The button contains the magnifying glass.